static/app/views/insights/common/components/fullSpanDescription.tsx

```diff
@@ -134,7 +132,7 @@ function QueryClippedBox({group, children}: TruncatedQueryClipBoxProps) {
         icon: <IconOpen />,
         onClick: () =>
           navigate({
-            pathname: `${location.pathname}spans/span/${group}/`,
+            pathname: `insights/database/spans/span/${group}/`,
```

Comment:
I am concerned about using relative path, shouldn't this be like /organization/{org.slug}/insights/database or whatever?

Reply:
in fact, can you just use `useModuleURL('db')`

Reply:
Good call on `useModuleURL` but for my own future reference, what's the concern with the relative path?